test/nodes/test_document_classifier.py

```diff
@@ -22,6 +22,21 @@ def test_document_classifier(document_classifier):
         assert doc.to_dict()["meta"]["classification"]["label"] == expected_labels[i]
 
 
+@pytest.mark.integration
+def test_document_classifier_details(document_classifier):
+
+    docs = [
+        Document(content="""That's good. I like it.""", meta={"name": "0"}, id="1"),
+        Document(content="""That's bad. I don't like it.""", meta={"name": "1"}, id="2"),
+    ]
+    results = document_classifier.predict(documents=docs)
+    for doc in results:
+        meta_classification = doc.to_dict()["meta"]["classification"]
+        assert "details" in meta_classification
+        top_k = 2
+        assert len(meta_classification["details"]) == top_k
```

Comment:
This can be slightly simplified:
```suggestion
        assert "details" in doc.meta["classification"].keys()
        assert len(doc.meta["classification"]["details"]) == 2  # top_k = 2
```